src/scripts.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+let initialState = {
+  items: ['Apple', 'Bread', 'Carrot', 'Dumplings', 'Eggs', 'Fish', 'Garlic', 'Honey', 'Ice cream', 'Jam'],
+  itemMove: '',
+  index: null
+};
+
+Array.prototype.move = function (index, step) { 
+  let next = this[index + step];
+  this[index + step] = this[index];
+  this[index] = next;
+  return this
+};
+
+function getNextState(state=initialState, action) {
+  switch (action.type) {
+    case 'order':
+      return {
+        ...state,
+        itemMove: action.itemMove,
+        index: state.items.indexOf(action.itemMove)
+      }
+    case 'up':
+      return {
+        ...state,
+        items: state.items.move(state.index, -1),
+        index: state.items.indexOf(state.itemMove),
+      };
+    case 'down':
+      return {
+        ...state,
+        items: state.items.move(state.index, 1),
+        index: state.items.indexOf(state.itemMove),   
+      };
+    default:
+      return state;
+  }
+}
+
+const store = Redux.createStore(getNextState, initialState);
+
+document.getElementById('up').addEventListener('click', (event) => {
```

Comment:
you declared a variable for this button on row 73, 74.
move them up and use here